### PR TITLE
Update /download/iot for core18

### LINKED ISSUE #4524: /download/iot - Core 18 update
Update /download/iot to match the [copy doc](https://docs.google.com/document/d/1Sxdl5cGGe1wbHmKB2eL2zQLfVW3KBea02a2FIGx3VdQ/edit)

### PR DESCRIPTION
**Pull request by Greggless**
## Done

- Updated `/download/iot` to reflect Core18 updates. 
- Updated `/download/iot/<boardname>` to correct UC18 links
- Added `/installation-medias` page to keep users drifting to outdated content on `developer.ubuntu.com`
- updated `flash_image` include to redirect to new `/installation-medias` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)

## Issue / Card

Fixes #4524 

## Screenshots
image

### CODE REVIEW

**Comment by nottrobin** [comment]
Should this be marked Work In Progress until the child pages exist?

**Comment by Greggless** [comment]
Yes

**Comment by caldav** [comment]
@Greggless this is missing the KVM sentence under featured boards. 
"To run Ubuntu Core on your workstation, you can [install Ubuntu Core in KVM](/download/iot/kvm)."

File: templates/download/iot/index.html [hunk 1]
```diff
@@ -8,102 +8,77 @@
...
+<section class="p-strip">
+  <div class="row">
+    <h3>Other releases</h3>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <table class="">
+          <tr>
+            <th scope="row"><strong>Raspberry Pi CM3</strong></th>
+            <td><a href="/raspberry-pi-compute-module-3">Install Ubuntu Core 16 ›</a></td>
```
Comment (caldav) [inline templates/download/iot/index.html]:
```suggestion
             <td><a href="/download/iot/raspberry-pi-compute-module-3">Install Ubuntu Core 16 ›</a></td>
```

File: templates/download/iot/index.html [hunk 2]
```diff
@@ -8,102 +8,77 @@
...
+            <td></td>
+          </tr>
+          <tr>
+            <th scope="row"><strong>UP2</strong></th>
+            <td><a href="/up-squared-iot-grove-server">Install Ubuntu Server ›</a></td>
```
Comment (caldav) [inline templates/download/iot/index.html]:
```suggestion
             <td><a href="/download/iot/up-squared-iot-grove-server">Install Ubuntu Server ›</a></td>
```

File: templates/download/iot/index.html [hunk 3]
```diff
@@ -8,102 +8,77 @@
...
+            <td></td>
+          </tr>
+          <tr>
+              <th scope="row"><strong>Samsung Artik</strong></th>
+              <td><a href="/samsung-artick-5-10-server">Install Ubuntu Server ›</a></td>
```
Comment (caldav) [inline templates/download/iot/index.html]:
```suggestion
               <td><a href="/download/iot/samsung-artik-5-10-server">Install Ubuntu Server ›</a></td>
```

File: templates/download/iot/index.html [hunk 4]
```diff
@@ -8,102 +8,77 @@
...
+              <td class="u-align--right"><a href="/samsung-artick-5-10">Install Ubuntu Core 16 ›</a></td>
```
Comment (caldav) [inline templates/download/iot/index.html]:
```suggestion
               <td class="u-align--right"><a href="/download/iot/samsung-artik-5-10">Install Ubuntu Core 16 ›</a></td>
```

File: templates/download/iot/index.html [hunk 5]
```diff
@@ -8,102 +8,77 @@
...
+            </tr>
+            <tr>
+                <th scope="row"><strong>Intel Joule</strong></th>
+                <td><a href="/intel-joule-desktop">Install Ubuntu Desktop ›</a></td>
```
Comment (caldav) [inline templates/download/iot/index.html]:
```suggestion
                 <td><a href="/download/iot/intel-joule-desktop">Install Ubuntu Desktop ›</a></td>
```

File: templates/download/iot/index.html [hunk 6]
```diff
@@ -8,102 +8,77 @@
...
+                <td class="u-align--right"><a href="/intel-joule">Install Ubuntu Core 16 ›</a></td>
```
Comment (caldav) [inline templates/download/iot/index.html]:
```suggestion
                 <td class="u-align--right"><a href="/download/iot/intel-joule">Install Ubuntu Core 16 ›</a></td>
```

File: templates/download/iot/index.html [hunk 7]
```diff
@@ -8,102 +8,77 @@
...
+              </tr>
+            <tr>
+                <th scope="row"><strong>Intel NUC</strong></th>
+                <td><a href="iot/intel-nuc-desktop">Install Ubuntu Desktop ›</a></td>
```
Comment (caldav) [inline templates/download/iot/index.html]:
```suggestion
                 <td><a href="/download/iot/intel-nuc-desktop">Install Ubuntu Desktop ›</a></td>
```

File: templates/download/iot/raspberry-pi-2-3.html [hunk 8]
```diff
@@ -49,18 +49,12 @@ <h3 class="p-list-step__title">
...
               </li>
-              <li>Raspberry Pi 3
-                <ul>
-                  <li><a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/16/stable/current/ubuntu-core-16-pi3.img.xz">Ubuntu Core 16 image for Raspberry Pi 3 (stable)</a> - the older kernel shipped with this image can trigger Wi-Fi issues during first boot, a network cable is recommended.</li>
-                  <li><a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/16/edge/current/ubuntu-core-16-armhf+raspi3.img.xz">Ubuntu Core 16 image for Raspberry Pi 3 (edge)</a> - this image provides reliable Wi-Fi at first boot, but is a daily build and not deemed stable.</li>
-                </ul>
+              <li>Raspberry Pi 3 • <a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/ubuntu-core-18-armhf+raspi3.img.xz">Download Ubuntu Core 18 image</a> - the older kernel shipped with this image can trigger Wi-Fi issues during first boot, a network cable is recommended.
```
Comment (caldav) [inline templates/download/iot/raspberry-pi-2-3.html]: This statement needs to be verified. Checking with Ogra.

Comment (caldav) [reply templates/download/iot/raspberry-pi-2-3.html]: We'll get a reply during the week, not a blocker.

File: templates/download/iot/index.html [hunk 9]
```diff
@@ -8,100 +8,77 @@
 <section class="p-strip is-bordered">
   <div class="row">
     <h1>Get Ubuntu for IoT</h1>
-    <p>Ubuntu is supported on a variety of chipsets, boards, SOCs and virtual machines, including Raspberry Pi 2 and 3, Compute Module 3, Qualcomm DragonBoard 410c, Intel NUC, Intel Joule, Samsung Artik and KVM.</p>
-    <p>We will guide you through the steps of installing Ubuntu Core, Ubuntu Desktop, or Ubuntu Server on these platforms.</p>
+    <p>Ubuntu Core is a tiny, transactional version of Ubuntu for IoT devices and large container deployments. It runs a new breed of super-secure, remotely upgradeable Linux app packages known as snaps ‐it’s trusted by leading IoT players, from chipset vendors to device makers and system integrators.</p>
```
[the remark this answers is not in the document]
Comment (Greggless) [reply templates/download/iot/index.html]: Thanks @caldav 

I used the copy from the old /core page. But i'm not sure we want to do that. Here's a version that might work a little better (its shorter too):
> Ubuntu Core is the most secure embedded Linux for devices, boards, and connected things. It runs snaps - super-secure, remotely upgradeable linux packages.

[the remark this answers is not in the document]
Comment (caldav) [reply templates/download/iot/index.html]: wfm

File: navigation.yaml [hunk 10]
```diff
@@ -327,6 +327,8 @@ download:
         - title: UP Squared IoT Grove
           path: /download/iot/up-squared-iot-grove-server
           hidden: True
+          title: Installation medias
```
Comment (nottrobin) [inline navigation.yaml]: This presumably needs a dash to denote that it's a new item in the list:

```suggestion

         - title: Installation media
```

File: templates/download/iot/index.html (hunk 9 above)
[the remark this answers is not in the document]
Comment (nottrobin) [reply templates/download/iot/index.html]: Does this page have a copy doc? I assume it should?

File: templates/download/iot/index.html [hunk 11]
```diff
@@ -8,100 +8,77 @@
...
+    <p><a href="/core">Learn more about Ubuntu Core ›</p><p> <a href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/" class="p-link--external">Browse all supported images</a></p>
```
Comment (nottrobin) [inline templates/download/iot/index.html]: You shouldn't put two paragraphs on one line.

```suggestion
    <p><a href="/core">Learn more about Ubuntu Core ›</p>
    <p><a href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/" class="p-link--external">Browse all supported images</a></p>
```

File: templates/download/iot/installation-medias.html [hunk 12]
```diff
@@ -0,0 +1,239 @@
+{% extends "download/_base_download.html" %}
```
Comment (nottrobin) [inline templates/download/iot/installation-medias.html]: This page, "installation-medias.html" should presumably be named "installation-media.html". The word "media" is already a plural. ["Medium"](https://en.oxforddictionaries.com/definition/medium) is the singular.

File: templates/download/iot/installation-medias.html [hunk 13]
```diff
@@ -0,0 +1,239 @@
...
+
+{% block title %}Create installation medias for Ubuntu Core{% endblock %}}
```
Comment (nottrobin) [inline templates/download/iot/installation-medias.html]:
```suggestion
{% block title %}Create installation media for Ubuntu Core{% endblock %}}
```

File: templates/download/iot/installation-medias.html [hunk 14]
```diff
@@ -0,0 +1,239 @@
...
+{% block meta_copydoc %}{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <div>
+        <h1>Create installation medias for Ubuntu Core</h1>
```
Comment (nottrobin) [inline templates/download/iot/installation-medias.html]:
```suggestion
         <h1>Create installation media for Ubuntu Core</h1>
```

File: navigation.yaml [hunk 15]
```diff
@@ -327,6 +327,8 @@ download:
...
+          path: /download/iot/installation-medias
```
Comment (nottrobin) [inline navigation.yaml]: Should be "installation-media".

```suggestion
          path: /download/iot/installation-media
```

File: templates/download/iot/_flash-image.html [hunk 16]
```diff
@@ -4,6 +4,6 @@ <h3 class="p-list-step__title">
     Flash the {{ card|default:"card" }}
   </h3>
   <div class="p-list-step__content">
-    <p>Copy the Ubuntu Core image on the {{ card|default:"card" }} by following the <a class="p-link--external" href="https://developer.ubuntu.com/core/get-started/installation-medias">installation media instructions</a>.</p>
+    <p>Copy the Ubuntu Core image on the {{ card|default:"card" }} by following the <a class="p-link" href="/download/iot/installation-medias">installation media instructions</a>.</p>
```
Comment (nottrobin) [inline templates/download/iot/_flash-image.html]:
```suggestion
    <p>Copy the Ubuntu Core image on the {{ card|default:"card" }} by following the <a class="p-link" href="/download/iot/installation-media">installation media instructions</a>.</p>
```

File: navigation.yaml [hunk 17]
```diff
@@ -328,6 +328,9 @@ download:
           path: /download/iot/up-squared-iot-grove-server
           hidden: True
 
+         - title: Installation media
```
Comment (nottrobin) [inline navigation.yaml]: Sorry, this was clearly my fault, and broke the test. Should be:

```suggestion
         - title: Installation media
```

File: navigation.yaml [hunk 18]
```diff
@@ -328,6 +328,9 @@ download:
...
+          path: /download/iot/installation-media
```
Comment (nottrobin) [inline navigation.yaml]: You probably also want `hidden: True` like all the others:

```suggestion
           path: /download/iot/installation-media
           hidden: True
```

File: templates/download/iot/index.html (hunk 9 above)
[the remark this answers is not in the document]
Comment (Greggless) [reply templates/download/iot/index.html]: It does

**Comment by Greggless** [comment]
@nottrobin `code +1` ?

**Comment by Greggless** [comment]
# 🐶 🇲🇰 📟 🍾 
**TY** 💋

**Pull request merged**